### PR TITLE
First major documentation release

### PR DESCRIPTION
**Pull request by bdbell**
First major import.  While there have been some minor adjustments and fixes for this submission, the preview of what it will look like is here: https://bdbell.github.io/VoronDocs/

### CODE REVIEW

**Review by jdlongenecker — COMMENTED** [review]
Hello @bdbell I've done an initial review of most of the text for this pull request. Great job! I haven't reviewed the links/flow and some things like that, but the text itself looks good.

File: build/electrical/mini12864_klipper_guide.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+1. Pry off the two black connector housings from the back of the display
+2. Rotate connector housings 180 degrees and insert back onto pins
+3. Connect EXP1 and EXP2 cables to the display and to your MCU board, being sure to match port numbers (EXP1 <> EXP1).
+
+## Klipper
+
+For SKR boards, add the following `[board_pins]` section to the configuration:
+
+```
+[board_pins]
```
[the remark this answers is not in the document]
Comment (bdbell) [reply build/electrical/mini12864_klipper_guide.md:25]: Not a problem, but the whole configuration will need to be updated then as the pins are referenced farther down.

File: build/software/flyf407zg_klipper.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* Klipper must be installed onto the Raspberry Pi
+* At least one microSD card needs to be available depending on number of controllers.
+
+### Build Firmware Image
+
+* Login to the Raspberry Pi
+* Run the following:
+
+```
+sudo apt install make
```
[the remark this answers is not in the document]
Comment (bdbell) [reply build/software/flyf407zg_klipper.md:20]: I believe this step is required if make is not installed.

File: build/startup/index.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+* Clear any stored bed meshes with `BED_MESH_CLEAR`
+
+Run `Z_ENDSTOP_CALIBRATE`
+
+Slowly move the nozzle toward the bed by using `TESTZ Z=-1`
+Until the nozzle is relatively close to the bed, and then stepping down with `TESTZ Z=-0.1`
+Until the nozzle touches a piece of paper on top of the build plate. If you go far down, you can move the nozzle back up with: `TESTZ Z=0.1`
+Once you are satisfied with the nozzle height, run `ACCEPT` and then `SAVE_CONFIG`.
+
+**Important:** Klipper assumes that this process is being done cold.  If being performed hot, do an additional `TESTZ Z=-0.1` before accepting.
```
[the remark this answers is not in the document]
Comment (bdbell) [reply build/startup/index.md:285]: The Klipper documentation (https://www.klipper3d.org/Bed_Level.html) explicitly calls out to do the calibration at room temperature at is assumes that there will be approximately 0.1mm of thermal expansion.

File: build/software/skr13_klipper.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* Klipper must be installed onto the Raspberry Pi
+* At least one microSD card needs to be available depending on number of controllers.
+
+### Build Firmware Image
+
+* Login to the Raspberry Pi
+* Run the following:
+
+```
+sudo apt install make
```
[the remark this answers is not in the document]
Comment (bdbell) [reply build/software/skr13_klipper.md:22]: This step is needed if make is not already installed.  The reference just just 'make' is further down.

**Comment by drayde** [comment]
I'm blown away by the quality of your documentation. Especially the mechanical assembly pdf. Just wow!
At the FabMX project (fabmx.org) we are currently designing/building a 3d printer as well (quite a different one) and I'm looking into ways of documenting the assembly. How do you make your assembly pdf? Is it "just" a lot of manual work, or is there some trickery that helps with creating all the images etc? Some insight would be very helpful. Thx

**Pull request merged**